.github/workflows/ci_api.yml

```diff
@@ -19,7 +19,6 @@ on:
 
 env:
   CI: "true"
-  RUBY_VERSION: 2.7.5
```

Comment:
We should define the target version for all the CI versions, right?

Reply:
Yes, it has been taken care of.